src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2107,8 +2110,8 @@ private void recordRelocation(IntPtr _this, void* location, void* target, ushort
             switch (targetBlock)
             {
                 case BlockType.Code:
-                    // TODO: Arbitrary relocs
-                    throw new NotImplementedException("Arbitrary relocs");
+                    reloc.Target = _methodCodeNode;
```

Comment:
I'm fine with the code in the PR. I prefer only keeping _methodCodeNode. This would also work: `_compilation.NodeFactory.MethodEntrypoint(_methodBeingCompiled);`